src/Controls/tests/TestCases.HostApp/Issues/Issue24246.xaml

```diff
@@ -0,0 +1,11 @@
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue24246"
+             Title="Issue24246"
+             xmlns:ios="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;assembly=Microsoft.Maui.Controls"
+             ios:Page.UseSafeArea="false">
+  <VerticalStackLayout Background="Purple" IgnoreSafeArea="False" VerticalOptions="Start" IsClippedToBounds="True" >
+    <Entry AutomationId="entry" Background="Green"></Entry>
+    <Label AutomationId="label" Text="If you can't interact with the above entry field, the test has failed" LineBreakMode="CharacterWrap"></Label>
+  </VerticalStackLayout>
+</ContentPage>
```

Comment:
![image](https://github.com/user-attachments/assets/7f62f5c2-ff84-449f-81f0-1c0e591cb74c)

Reply:
<img width="438" alt="image" src="https://github.com/user-attachments/assets/5d438a84-109e-4411-aed8-010013d0f38a" />

I was curious what this looked like on main so if anyone else was wondering :)
edit: this is already here: https://github.com/dotnet/maui/pull/28264